set.mm

```diff
@@ -584987,6 +584987,29 @@ dependency is expressed in our hypothesis (called implicit
     ( wi wsb wnf sbim sbft imbi2d bitrid ) ABECDFACDFZBCDFZEBCGZLBEABCDHNMBLBCD
     IJK $.
 
+  ${
+    $d x y $.
+    $( Commutation of quantification and substitution variables based on fewer
+       axioms than ~ sb9 .  (Contributed by Wolf Lammen, 27-Apr-2025.) $)
+    wl-sb9f $p |- ( A. x [ x / y ] ph <-> A. y [ y / x ] ph ) $=
```

Comment:
looks like sb9v

Reply:
See [sb8f](https://us.metamath.org/mpeuni/sb8f.html) and [sb8v](https://us.metamath.org/mpeuni/sb8v.html)

Reply:
I don't see a `F/` hypothesis like sb8f